tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -1400,29 +1371,38 @@ def test_continuous_writes_during_kpods_down(self):
         resp = self.ha_obj.check_s3bench_log(file_paths=fail_logs, pass_logs=False)
         assert_utils.assert_true(len(resp[1]) <= len(fail_logs),
                                  f"Logs which contain pass: {resp[1]}")
-        LOGGER.info("Step 6: Verified status for In-flight WRITEs while %s (K) pods "
+        LOGGER.info("Step 3: Verified status for In-flight WRITEs while %s (K) data pods "
                     "going down.", self.kvalue)
 
-        LOGGER.info("Step 7: Create multiple buckets and run IOs")
-        self.test_prefix = 'test-35776-1'
-        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
-                                                    log_prefix=self.test_prefix, nsamples=2,
-                                                    nclients=2)
+        if CMN_CFG["dtm0_disabled"]:
+            LOGGER.info("STEP 4: Create IAM user and perform WRITEs-READs-Verify-DELETEs with "
+                        "variable object sizes on degraded cluster")
+            users = self.mgnt_ops.create_account_users(nusers=1)
+            self.test_prefix = 'test-35776-1'
+            self.s3_clean.update(users)
+            resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                        log_prefix=self.test_prefix,
+                                                        nsamples=2, nclients=2)
+        else:
+            LOGGER.info("STEP 4: Perform WRITEs-READs-Verify with variable object sizes on "
+                        "degraded cluster")
+            resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                        log_prefix=self.test_prefix,
+                                                        skipcleanup=True, nsamples=2, nclients=2)
         assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Step 7: Successfully created multiple buckets and ran IOs")
-        LOGGER.info("ENDED: Test to verify continuous WRITEs during %s (K) data pods down",
-                    self.kvalue)
+        LOGGER.info("Step 4: Performed IOs with variable sizes objects.")
+
+        LOGGER.info("ENDED: Test to verify WRITEs during %s (K) data pods down", self.kvalue)
 
-    # pylint: disable=C0321
-    # pylint: disable=too-many-locals
     @pytest.mark.ha
     @pytest.mark.lc
+    @pytest.mark.skip(reason="Buckets cruds won't be supported with DTM0")
     @pytest.mark.tags("TEST-35780")
     def test_deletes_during_kpods_down(self):
         """
-        This test tests continuous DELETEs while pods are failing till K data pods are failed
+        This test tests DELETEs while pods are failing till K data pods are failed
```

Comment:
Why remove continuous?

Reply:
> Why remove continuous?

while and continuous are both stating same things